tools/odbc/parameter_descriptor.cpp

```diff
@@ -3,6 +3,8 @@
 #include "duckdb/common/types/decimal.hpp"
 #include "handle_functions.hpp"
 
+#include <iostream>
+
```

Comment:
Nit: Is this neccisary?

Reply:
Sorry left over from adding cout statements to driver.